release_templates/mxnet_release_images.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Let's make release_templates folder hidden by default on the filesystem: `.release_templates`, or create `.templates/release`.

Reply:
Why?